### DISCUSSION TITLE
[Compatibility] I have a question to using yarn berry about install to yarn/sdks for IntelliJ IDE from JetBrains.

### DISCUSSION THREAD

**Discussion by willofi** [yarnpkg/berry#6378] [Q&A]
First of all, Thank you for your help and contributing during my develop career.  
I wanna use yarn berry (zero-install) with IntelliJ IDE from JetBrains already mentioned with title.  
But, i can't find that how install Specific Editor's SDK.

https://yarnpkg.com/getting-started/editor-sdks

Upper page is only shown to vscode IDE.. sadly.
May I know how can i use yarn berry with IntelliJ?

Thanks.

**Comment by clemyan** [comment]
As far as I know, IntelliJ supports PnP out-of-the-box
